spec/mongo/index/view_spec.rb

```diff
@@ -196,6 +196,47 @@
         expect(view.get('random_name')).to_not be_nil
       end
     end
+
```

Comment:
You could do a structure with contexts like this instad:

context 'when providing a partial index filter', if: find_command_enabled? do
   context 'when the filter is invalid' do
   context 'when the filter is valid' do